mage_ai/server/api/orchestration.py

```diff
@@ -369,6 +369,17 @@ def get(self, pipeline_uuid=None):
                 collection = [r.to_dict(include_attributes=['event_matchers', 'pipeline_runs_count'])
                               for r in results]
                 collection.sort(key=lambda d: d['id'], reverse=True)
+
+                for schedule in collection:
+                    runs = (
```

Comment:
no need to do additional DB queries. you can do `schedule.pipeline_runs` to get all pipeline runs.
You can sort them with the code like
```
sorted(ps.pipeline_runs, key=lambda x: x.created_at)
```

Reply:
thank you xiaoyou!